### ISSUE TITLE
No view is registered with id: zowe.spool

### ISSUE THREAD

**Issue by ChrisBoehmCA** [zowe/vscode-extension-for-zowe#130]
![image](https://user-images.githubusercontent.com/42545737/60194305-0eae2e80-9807-11e9-9569-88aff188bded.png)

Since updating to 0.20.0 ( I believe, hadn't opened the extension in a while), I get this error on startup and can't view job spool anywhere anymore

Tried reinstalling but that didn't seem to help

**Comment by dbaldon** [comment]
I get the same error on 0.21.0. I reverted to 0.19.0 and things worked again. Then I found this in the code which points to the CHANGELOG.md file which contains this;
0.20.0
Combined Spool files with Jobs in Jobs view. Colin Stone

So I installed 0.20. and things work as expected. Release 0.19.0 had the separate spool section and 0.21.0 doesn't present the combined view nor does it have the separate spool section and throws the error message shown in the previous post. Looks like a code change got dropped between 0.20.0 and 0.21.0.
